content/en/docs/languages/java/api.md

```diff
@@ -600,7 +600,7 @@ public class OpenTelemetryUsage {
 ```
 <!-- prettier-ignore-end -->
 
-### GlobalOpenTelemetry
+#### GlobalOpenTelemetry
```

Comment:
I'm not sure about the organization of the additions in this file

Maybe could just add two small notes right above here, something like "If you are using the Java agent, see X for information about how to obtain an OpenTelemetry instance" (and similar note for Spring Boot starter)

Reply:
I agree - I think an alert could be good. Something like
```
{{% alert title="Spring Boot Starter" %}} 
The Spring Boot starter is a special case where [...].
{{% /alert %}}
```